src/js/helpers/GoogleAnalyticsPageView.js

```diff
@@ -1,6 +1,8 @@
 import ReactGA from 'react-ga'
 
 const GoogleAnalyticsPageView = () => {
+  if (typeof document === 'undefined') return false;
```

Comment:
@ifsimicoded, This bombs in static build mode because window is undefined. But  loads progressively in the built pages. Is this a problem at all?

Reply:
your change looks good. the React GA library uses `typeof window === 'undefined'`
https://github.com/react-ga/react-ga/blob/3d08ae2fbccf0209ae827a7f0128843301a3eff8/src/index.js#L81
Honestly, I don't know what's better -- the type check for undefined in a stateless functional component, or putting code you want to run on the client only in a ComponentDidMount function. The latter is how the Recollect script loads. I can update the Recollect component to be consistent.

Reply:
Seems like doing the document check is the best thing to do in a stateless function and putting this type of function in ComponentDidMount is the best thing to do if the component has state. Consistency is good, but I don't think we have enough examples to refactor towards "the best" pattern for these two different types of components.